event-management-frontend/src/styles/OccupancyCardsStyle.tsx

```diff
@@ -1,90 +1,121 @@
-import { makeStyles } from "@material-ui/core/styles";
-import themeDark from "./Apptheme";
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Here you should align with your colleagues, either use single quotes or double quotes for the imports. Otherwise such swap situations will continuosly occur